### ISSUE TITLE
Didn't find op for builtin opcode 'SOFTMAX' version '1' error on Cortex-M7

### ISSUE THREAD

**Issue by alisonwh** [tensorflow/tensorflow#45332]
@tensorflow/micro

**System information**
- Host OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 18.04
- TensorFlow installed from (source or binary): source
- Target platform (e.g. Arm Mbed OS, Arduino Nano 33 etc.): i.MX8MP's Cortex-M7 core

**Describe the problem**
I am running tflite micro on i.MX8MP's Cortex-M7 core with mobilenet_v1_1.0_224_quant.tflite. I also enable cmsis-nn in the building. According to the following log, the operators CONV_2D, DEPTHWISE_CONV_2D, SOFTMAX.. are all registered. But SOFTMAX isn't found. It is the last operator for the loop in PrepareNodeAndRegistrationDataFromFlatbuffer().

My test code is very simple,

const tflite::Model *model = ::tflite::GetModel(job.networkModel.data);
tflite::AllOpsResolver micro_op_resolver;
 
tflite::MicroInterpreter interpreter(model, micro_op_resolver, inferenceProcessTensorArena, TENSOR_ARENA_SIZE, reporter);
TfLiteStatus allocate_status = interpreter.AllocateTensors();
...

Log:
..
Register cmsis-nn CONV_2D
Register cmsis-nn DEPTHWISE_CONV_2D
Register Kernel common RESHAPE
Register cmsis-nn SOFTMAX
..
micro_interpreter.cc AllocateTensors
micro_allocator.cc: subgraph->operators()->size() 31
..
micro_allocator.cc: PrepareNodeAndRegistrationDataFromFlatbuffer i: 30
micro_allocator.cc: To run GetRegistrationFromOpCode for op code SOFTMAX
op_solver.cc: GetRegistrationFromOpCode
Didn't find op for builtin opcode 'SOFTMAX' version '1'
Failed to get registration from op code SOFTMAX
 
AllocateTensors failed for inference job: job, status 1

**Comment by alisonwh** [comment]
Found the root cause and close it.
